spec/features/registration/change_password.spec.rb

```diff
@@ -0,0 +1,30 @@
+describe "Change password", type: :feature do
+  let(:user) { create(:user) }
+
+  before do
+    sign_in_user user
+  end
+
+  it_behaves_like "shows the settings page"
+
+  context "when changing a password" do
+    it "displays the change password link on the settings page" do
+      visit settings_path
+      expect(page).to have_content("Change your password")
+    end
+
+    it "takes you to the form to change your password" do
+      click_on "Change your password"
+      expect(page).to have_content("Change your password")
+    end
+
+    it "displays accordion with text 'help with choosing a strong password'" do
+      expect(page).to have_content("Help with choosing a strong password")
+    end
+
+    it "displays clear instructions on help with choosing a strong password" do
+      click_on "Help with choosing a strong password"
```

Comment:
the design system element that expands the help section is based on Javascript, which is not enabled in these tests. Seeing how it is from a library maybe we don't need to test that functionality?